Services/DishesService.js

```diff
@@ -0,0 +1,16 @@
+const ObjectId = require('mongodb').ObjectId
+
+let getAllDishes = async (db, req) => {
+    const collection = db.collection('dishes')
+    const data = await collection.find({}).toArray()
+    return data
+}
+
+let getAllDishesOfType = async (db, req) => {
+    const collection = db.collection('dishes')
+    const data = await collection.find({dishType: req.params.course}).toArray()
```

Comment:
I'd suggest passing in just the course here rather than the whole request object